driver/stub/stub_usbd.c

```diff
@@ -200,16 +200,18 @@ get_usb_dsc_conf(usbip_stub_dev_t *devstub, UCHAR bVal)
 	USB_CONFIGURATION_DESCRIPTOR	ConfDesc;
 	PUSB_CONFIGURATION_DESCRIPTOR	dsc_conf;
 	ULONG	len;
-
-	if (!find_usb_dsc_conf(devstub, bVal, &ConfDesc))
+	INT   iConfiguration;
+	
+	iConfiguration = find_usb_dsc_conf(devstub, bVal, &ConfDesc);
+	if (iConfiguration == -1)
 		return NULL;
 
 	dsc_conf = ExAllocatePoolWithTag(NonPagedPool, ConfDesc.wTotalLength, USBIP_STUB_POOL_TAG);
 	if (dsc_conf == NULL)
 		return NULL;
 
 	len = ConfDesc.wTotalLength;
-	if (!get_usb_desc(devstub, USB_CONFIGURATION_DESCRIPTOR_TYPE, ConfDesc.iConfiguration, 0, dsc_conf, &len)) {
+	if (!get_usb_desc(devstub, USB_CONFIGURATION_DESCRIPTOR_TYPE, (UCHAR)iConfiguration, 0, dsc_conf, &len)) {
```

Comment:
Yes, you're right. I didn't know that iConfiguration is an index of string descriptor for a configuration.
This is the cause of the problem.